docs/content/en/docs/reference/clusterspec/baremetal.md

```diff
@@ -12,13 +12,576 @@ The following additional optional configuration can also be included:
 * [CNI]({{< relref "optional/cni.md" >}})
 * [multus]({{< relref "optional/multus.md" >}})
 
+To generate your own cluster configuration, follow instructions from the Bare Metal [Create production cluster]({{< relref "../../getting-started/production-environment/" >}}) section and modify it using descriptions below.
+For information on how to add cluster configuration settings to this file for advanced node configuration, see [Advanced Bare Metal cluster configuration]({{< relref "#advanced-bare-metal-cluster-configuration" >}}).
+
 
 ```yaml
+apiVersion: anywhere.eks.amazonaws.com/v1alpha1
+kind: Cluster
+metadata:
+  name: my-cluster-name
+spec:
+  clusterNetwork:
+    cniConfig:
+      cilium: {}
+    pods:
+      cidrBlocks:
+      - 192.168.0.0/16
+    services:
+      cidrBlocks:
+      - 10.96.0.0/12
+  controlPlaneConfiguration:              
+    count: 1
+    endpoint:
+      host: "193.17.0.50"
```

Comment:
Instead of hardcoding a random IP here, can we either leave it empty or say 
```suggestion
      host: "<Control Plane Endpoint IP>"
```

Reply:
Done.